Formula/mariadb-connector-odbc.rb

```diff
@@ -0,0 +1,25 @@
+class MariadbConnectorOdbc < Formula
+  desc "MariaDB Standardized database driver"
```

Comment:
seems a bit vague

Reply:
Copied a part of the description from the homepage.  :wink: